### PR TITLE
PPTP-1566 Confirm / Capture Group member Contact Address (UK only)

### PR DESCRIPTION
**Pull request by kiranprakashdigital**
### Description of Work carried through

Brief description of what/why/how.

#### Check list 
 - [x] `./precheck` was executed (Integration/Component/Unit tests)
 - [x] `sbt scalafmt test:scalafmt` was executed
 - [x] User Acceptance Tests (UAT) were run locally and have passed.
 - [x] No Accessibility errors/warnings reported by Wave

### CODE REVIEW

File: test/uk/gov/hmrc/plasticpackagingtax/registration/controllers/group/ConfirmContactAddressControllerSpec.scala [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+import play.api.test.Helpers.{await, redirectLocation, status}
+import uk.gov.hmrc.plasticpackagingtax.registration.connectors.addresslookup.AddressLookupFrontendConnector
+import uk.gov.hmrc.plasticpackagingtax.registration.controllers.group.{routes => groupRoutes}
+import uk.gov.hmrc.plasticpackagingtax.registration.forms.contact.Address
+import uk.gov.hmrc.plasticpackagingtax.registration.models.addresslookup._
+import uk.gov.hmrc.play.bootstrap.tools.Stubs.stubMessagesControllerComponents
+
+import scala.concurrent.Future
+
+class ConfirmContactAddressControllerSpec extends ControllerSpec {
```
Comment (tmc-ee) [inline test/uk/gov/hmrc/plasticpackagingtax/registration/controllers/group/ConfirmContactAddressControllerSpec.scala:33]: Do we need to add any tests for failed requests in this file? e.g. ALF can't be reached/returns an error code?

File: app/uk/gov/hmrc/plasticpackagingtax/registration/controllers/group/ContactDetailsTelephoneNumberController.scala [hunk 2]
```diff
@@ -85,10 +86,10 @@ class ContactDetailsTelephoneNumberController @Inject() (
                 )
               )
             ),
-          emailAddress =>
-            updateRegistration(emailAddress).map {
+          phoneNumber =>
+            updateRegistration(phoneNumber).map {
               case Right(_) =>
-                Redirect(groupRoutes.OrganisationListController.displayPage())
+                Redirect(groupRoutes.ConfirmContactAddressController.displayPage())
```
Comment (RyanKellyHMRC) [inline app/uk/gov/hmrc/plasticpackagingtax/registration/controllers/group/ContactDetailsTelephoneNumberController.scala:92]: You might want to consider hooking in the new group member CYA page which I've just merged here.

**Review by RyanKellyHMRC — APPROVED** [review]
All good. You might want to hook in the new group member CYA page whilst fixing the merge conflicts?

File: app/uk/gov/hmrc/plasticpackagingtax/registration/controllers/group/ContactDetailsTelephoneNumberController.scala (hunk 2 above)
Comment (kiranprakashdigital) [reply app/uk/gov/hmrc/plasticpackagingtax/registration/controllers/group/ContactDetailsTelephoneNumberController.scala]: Done

File: test/uk/gov/hmrc/plasticpackagingtax/registration/controllers/group/ConfirmContactAddressControllerSpec.scala (hunk 1 above)
Comment (kiranprakashdigital) [reply test/uk/gov/hmrc/plasticpackagingtax/registration/controllers/group/ConfirmContactAddressControllerSpec.scala]: These are handled in address lookup connector, just handling MissingAddressIdException

**Review by tmc-ee — APPROVED** [review]

**Pull request merged**